diff --git a/app/build.gradle b/app/build.gradle
index 4ad7f8cd..a61bd786 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -5,8 +5,6 @@ buildscript {
     }
 
     dependencies {
-        classpath 'io.fabric.tools:gradle:1.+'
-        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.4'
         classpath 'de.undercouch:gradle-download-task:3.4.3'
     }
 }
@@ -14,14 +12,9 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.google.gms.google-services'
-apply plugin: 'io.fabric'
-apply plugin: 'jacoco-android'
 apply plugin: 'de.undercouch.download'
 
 
-jacoco {
-    toolVersion = "0.8.3"
-}
 
 ext {
     supportLibraryVersion = "28.0.0"
diff --git a/build.gradle b/build.gradle
index 285afb4f..cf7c5f30 100644
--- a/build.gradle
+++ b/build.gradle
@@ -10,7 +10,6 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.6.1'
         classpath 'com.google.gms:google-services:4.3.3'
-        classpath 'io.fabric.tools:gradle:1.31.2'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
diff --git a/wear/build.gradle b/wear/build.gradle
index 3a2277e2..e7c7957a 100644
--- a/wear/build.gradle
+++ b/wear/build.gradle
@@ -4,15 +4,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.4'
     }
 }
 apply plugin: 'com.android.application'
-apply plugin: 'jacoco-android'
 
-jacoco {
-    toolVersion = "0.8.3"
-}
 
 ext {
     wearableVersion = "2.4.0"
